content/articles/running-a-multi-container-springboot-postgresql-application-with-docker-compose/index.md

```diff
@@ -0,0 +1,207 @@
+Container technology has streamlined how we build, test and deploy software from our local environment to on-premise data centers and the cloud. With the benefit of building applications with container technology, manually starting and stopping each container becomes tedious as you build multi-container applications. 
+
+Docker Compose is a tool used in defining and running multi-container Docker applications. We use Docker Compose to replace the `run` commands that is repeatedly run on each container in a multi-container application. Using Docker Compose, we can simplify the running of multi-container applications to as little as two commands; `docker-compose up` and `docker-compose down`.
+
+### Goal
+
+In this tutorial, you’ll learn how Docker Compose makes it easier to configure and run multiple Docker containers in a local environment. For demo purposes, this tutorial will use a Spring Boot and a PostgreSQL container.
+
+### Prerequisites
+
+- Basic knowledge of Spring Boot, Git, and the terminal.
+- Basic understanding of Docker and Docker Compose installed - see guide [here](https://docs.docker.com/compose/install/).
+- A Java IDE - In this tutorial, we will use [IntelliJ Idea](https://www.jetbrains.com/idea/), but you can use any IDE of your choice.
+- Postman - download it [here](https://www.postman.com/downloads/)
```

Comment:
Please add a single sentence explaining what Postman is before the "download it here" section. This is beneficial to a reader

Reply:
Noted.

Reply:
Done. Do check it out. Made the right edit in the last commit.